.gitignore

```diff
@@ -3,6 +3,9 @@ logs
 *.log
```

Comment:
This gitignore is decent however it would ignore certain things like .DS_Store which should not be a part of your code base.. also they can cause merge conflicts